ui_framework/doc_site/src/views/tool_bar/tool_bar_search_only.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import {
+  KuiToolBar,
+  KuiToolBarSearchBox,
+} from '../../../../components';
+
+export const ToolBarSearchOnly = () => (
+  <KuiToolBar className="kuiToolBar--searchOnly">
```

Comment:
To me `searchOnly` feels like "a wrong abstraction", and you get straight into the problem @Stacey-Gammon mentions. I think it's much better to turn on and off specific features, than having one that turns off lots of things. If this was `showSomeField: boolean` that defaulted to `true`, you could turn it off to have the equivalent of "search only". That feels like an abstraction that can "grow over time".